common/app/layout/Front.scala

```diff
@@ -170,7 +173,8 @@ object FaciaContainer {
     config: CollectionConfigWithId,
     collectionEssentials: CollectionEssentials,
     containerLayout: Option[ContainerLayout],
-    componentId: Option[String]
+    componentId: Option[String],
+    omitMPU: Boolean = false
```

Comment:
I've removed the default value from the case class now. This line is actually a default parameter value.